backend/domain/transaction/transaction_schema.py

```diff
@@ -0,0 +1,41 @@
+from pydantic import BaseModel, validator
+from models import Account
+from datetime import datetime
+
+
+class TransactionCreate(BaseModel):
+    transaction_date: datetime
+    transaction_amount: float
+
+    @validator("transaction_date", "transaction_amount")
+    def date_not_empty(cls, v):
+        if not v:
+            raise ValueError("This is a required field.")
+        return v
+
+
+class TransactionUpdate(BaseModel):
+    transaction_date: datetime
+    transaction_amount: float
+    account_id: str
```

Comment:
Please remove account_id and account from the schema. What will probably happen is the endpoint is going to be something like /{account_id}/{transaction_id} and you will fetch account from it in transaction.py. The update object itself should not have the account_id and account attributes.